datasets/us/navy/crawler.py

```diff
@@ -0,0 +1,129 @@
+from lxml import html, etree
+import orjson
+from urllib.parse import urljoin
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+from zavod.shed.zyte_api import fetch_html
+
+BASE_URL = "https://www.navy.mil"
+API_URL = "https://www.navy.mil/API/ArticleCS/Public/GetList?moduleID=709&dpage=%d&TabId=119&language=en-US"
+
+
+def emit_person(context: Context, country: str, source_url: str, role: str, name: str):
+    person = context.make("Person")
+    person.id = context.make_id(country, name, role)
+    person.add("name", name)
+    person.add("position", role)
+    person.add("sourceUrl", source_url)
+
+    position = h.make_position(context, role, country=country, topics=["mil"])
```

Comment:
For Position entities, we actually use gov.security https://www.opensanctions.org/docs/topics/#politically-exposed-persons - I think the thinking was to make it broad enough to cover police, intelligence, etc too.

```suggestion
    position = h.make_position(context, role, country=country)
```